### PR TITLE
added required trip type for transit passes

### PR DESCRIPTION
**Pull request by dmengelt**
TripType is mandatory: [trip-type-for-transit-passes](https://developers.google.com/wallet/tickets/transit-passes/qr-code/rest/v1/transitobject#TripType)

### CODE REVIEW

**Comment by dmengelt** [comment]
@nick-alteen I guess the samples for the different languages are generated. If not, let me know and I will adjust this pull request to also change all the other places.

File: java/src/main/java/DemoTransit.java [hunk 1]
```diff
@@ -384,7 +386,7 @@ public String CreateJWTSaveURL(String issuerId, String classSuffix, String userI
                             .setDefaultValue(
                                 new TranslatedString()
                                     .setLanguage("en-US")
-                                    .setValue("Fare name"))));
+                                    .setValue("Fare name")))));
```
Comment (ncalteen) [inline java/src/main/java/DemoTransit.java:389]: Did this compile correctly for you?

**Review by ncalteen — COMMENTED** [review]
Just one comment on the Java example

File: java/src/main/java/DemoTransit.java (hunk 1 above)
Comment (dmengelt) [reply java/src/main/java/DemoTransit.java:389]: No. Just checked. Thanks for the change 😀
Initially for Java I was setting the trip type at the [end of the object creation](https://github.com/google-pay/wallet-samples/commit/78570f4f4288e1e7fe18113c356986bc6f214766). After moving it I mistakenly left this brace.

**Comment by dmengelt** [comment]
fixed

**Pull request merged**